types/office-js-preview/index.d.ts

```diff
@@ -12023,6 +12023,9 @@ declare namespace Office {
         /**
          * Gets an item's categories.
          *
+         * **Important**: If there are no categories on the item, `null` or an empty array will be returned depending on the Outlook version
```

Comment:
Nit: I think there should be a comma between "version" and "so". (In all four repeated places.)

Reply:
Yeah, it could go either way. I'll leave it be for now.